### ISSUE TITLE
Error updating a table in a module

### ISSUE THREAD

**Issue by pangolp** [azerothcore/azerothcore-wotlk#16639]
### Current Behaviour

Trying to update the individual experience module, I have to modify one of the attributes, to convert it from int to float, so that it can take decimal values. When I process to create a sql file, inside the updates folder, the emulator closes, because it does not execute the content of the base folder. Therefore, when it arrives to the update query, it finds that the table does not exist, and therefore, it fails to update.

### Expected Blizzlike Behaviour

You should first folder all the content, which is inside the base folder, and then, move on to the updates. Because if not, we are going to have failures at the time of updating table attributes, renaming them, or any operation on them. At least of tables that already exist. If the same one does not exist, the error does not occur. 

### Steps to reproduce the problem

1. Take any community module
2. Create a SQL script in the updates folder, that modifies some attribute and it should close.

### Extra Notes

The tests were performed with the individual experience module, which is up to date, with the latest structure, so it should not fail.

### AC rev. hash/commit

https://github.com/azerothcore/azerothcore-wotlk/commit/a779cca9e3a3d673761d6bbe6d1c5d1f51c2f6f3

### Operating system

Windows 10

### Custom changes or Modules

mod-individual-xp

**Comment by heyitsbench** [comment]
I believe this is a duplicate of https://github.com/azerothcore/azerothcore-wotlk/issues/14766. Currently the core has no support for base/updates structured SQL queries for modules.

**Comment by pangolp** [comment]
> I believe this is a duplicate of #14766. Currently the core has no support for base/updates structured SQL queries for modules.

I did not know about this. By default, it should always run the emulator base queries first, I guess then its updates, and lastly, the module base and updates. Or I don't really know how the order should be, but without this, we can't update some of the modules. I was just trying to make, that the experience can be a value with comma, as we had discussed and I ran into this topic.
